code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -115,15 +134,17 @@
 	///A list of all the external organs we've got stored to draw horns, wings and stuff with (special because we are actually in the limbs unlike normal organs :/ )
 	var/list/obj/item/organ/external/external_organs = list()
 
-
 /obj/item/bodypart/Initialize(mapload)
 	. = ..()
 	if(can_be_disabled)
 		RegisterSignal(src, SIGNAL_ADDTRAIT(TRAIT_PARALYSIS), .proc/on_paralysis_trait_gain)
 		RegisterSignal(src, SIGNAL_REMOVETRAIT(TRAIT_PARALYSIS), .proc/on_paralysis_trait_loss)
-	if(status != BODYPART_ORGANIC)
+	if(!IS_ORGANIC_LIMB(src))
 		grind_results = null
 
+	name = "[limb_id] [parse_zone(body_zone)]"
```

Comment:
[![dreamseeker_2022-03-29_22-15-24.png](https://i.imgur.com/XMAEUmQl.jpg)](https://i.imgur.com/XMAEUmQ.png)

Changing the default name to the species + body_zone and explicitly ignoring the species if it's the same as the apparent species in human examine causes more side effects than the opposite of checking to include the notable species in human examine

Reply:
I changed item_attack() to use zone. I will apply this to wounds aswell.